cmd/prometheus/main.go

```diff
@@ -246,6 +249,12 @@ func main() {
 	a.Flag("query.max-samples", "Maximum number of samples a single query can load into memory. Note that queries will fail if they try to load more samples than this into memory, so this also limits the number of samples a query can return.").
 		Default("50000000").IntVar(&cfg.queryMaxSamples)
 
+	a.Flag("active.queries.filepath", "File that will log active queries").
```

Comment:
I don't think either of these need to be configurable.

Reply:
What would you like to be the defaults? I picked both of them at random: 100 queries at a time, and queries.log as filename.

Reply:
That's as good as anything.